polyply/tests/test_backmap.py

```diff
@@ -18,12 +18,31 @@
 from numpy.linalg import norm
 import networkx as nx
 import vermouth
+from vermouth.graph_utils import make_residue_graph
 from polyply import MetaMolecule
 from polyply.src.backmap import Backmap
 
 def test_backmapping():
-    meta_molecule = MetaMolecule()
-    meta_molecule.add_edges_from([(0, 1), (1, 2)])
+    """
+    Integral test for the backmapping processor.
+    """
+    molecule = vermouth.molecule.Molecule()
+    molecule.add_edges_from([(1, 2), (2, 3), (3, 4), (4, 5), (5, 6), (6, 7)])
+    nx.set_node_attributes(molecule, {
+        1: {"resname": "test", "resid": 1, "atomname": "A"},
+        2: {"resname": "test", "resid": 1, "atomname": "B"},
+        3: {"resname": "test", "resid": 1, "atomname": "C"},
+        4: {"resname": "test", "resid": 2, "atomname": "A"},
+        5: {"resname": "test", "resid": 2, "atomname": "B"},
+        6: {"resname": "test", "resid": 2, "atomname": "C"},
+        7: {"resname": "test", "resid": 3, "atomname": "C",
+            "position": np.array([4., 4., 4.])}
+        })
+
+    graph = make_residue_graph(molecule)
+    meta_molecule = MetaMolecule(graph)
+    meta_molecule.molecule = molecule
```

Comment:
Nit
```suggestion
    meta_molecule = MetaMolecule.from_molecule(molecule)
```

Reply:
Currently it doesn't have a from molecule method. I think we can do without for now and circle back later.

Reply:
I add a todo on the MetaMolecule